src/Makefile

```diff
@@ -177,7 +177,7 @@ cadical-download:
 	@$(TAR) xfz $(cadical_release).tar.gz
 	@rm -Rf ../cadical
 	@mv cadical-$(cadical_release) ../cadical
-	@cd ../cadical && CXX=$(CXX) ./configure -O3 -s && make -j
+	@cd ../cadical && ./configure -O3 -s CXX="$(CXX)" CXXFLAGS=-std=c++14 && make -j cadical
```

Comment:
We have so far not required C++14. This is a major change, and shouldn't sneak in with a PR like this.

Reply:
This was just for compiling cadical, and nothing else. IIRC, this was needed to compile cadical with clang, because without it, it fails on this [line](https://github.com/arminbiere/cadical/blob/master/src/lookahead.cpp#L426), which cannot be compiled with a pre-C++11 standard (right angle brackets required a space in between). I think it should work with `-std=c++11` too. Would this suffice?

Reply:
I changed it to `-std=c++11`.

Reply:
Sorry to be a pain here: please do the Cadical build using the Makefile system. In particular, ensure that the compilation options specified in the configuration are used.

Otherwise, anybody who wants to change any of the architectural settings has to find that line and change it there as well. We would prefer to enable architectural compiler flags to be given in a central place.

Reply:
Good point. Done.